frontend/src/components/collaborator/CollaboratorListItem.vue

```diff
@@ -1,104 +1,111 @@
 <template>
-  <v-list-item class="px-0" two-line>
-    <v-list-item-avatar>
-      <user-avatar :camp-collaboration="collaborator" />
-    </v-list-item-avatar>
+  <v-list-item class="px-0 e-collaborator-item" two-line>
+    <v-list-item-action>
+      <user-avatar size="40" :camp-collaboration="collaborator" />
+    </v-list-item-action>
     <v-list-item-content>
-      <v-list-item-title v-if="collaborator.user">
-        {{ collaborator.user().displayName }}
+      <v-list-item-title>
+        <span v-if="collaborator.user">{{ collaborator.user().displayName }}</span>
+        <span v-else>{{ collaborator.inviteEmail }}</span>
       </v-list-item-title>
-      <v-list-item-subtitle v-else>
-        {{ collaborator.inviteEmail }}
+      <v-list-item-subtitle>
+        {{ $tc(roles[collaborator.role].translation) }} &middot;
+        <template v-for="icon in roles[collaborator.role].icons">
+          <v-icon :key="icon" x-small>{{ icon }}</v-icon
+          >&nbsp;
+        </template>
```

Comment:
These icons represent the permissions which this person has in the camp, right? Are they really understandable this way? I had to think about it for a while until I got it.
Initially I thought these are clickable or mean separate things. Maybe you could reduce the spacing of the icons a little, so that it is clearer that they belong together..?

Reply:
Yes. I will fix this issue. Thanks for pointing out!

Sadly I can't test on non retina devices., I will try to look at the centering.

Reply:
You can use our browserstack account. I think I have invited you all previously, but let me know if you need another invite

Reply:
Or at least in Firefox, in responsive design mode, you can simulate DPRs other than your device:
![Screenshot from 2023-03-15 11-17-11](https://user-images.githubusercontent.com/7566995/225279834-4cd4b4b4-3808-4143-ae6f-bb763a5f1f2e.png)
But not sure how complete this simulation is...

Reply:
I would not rely on that. Also reviewing non retina content on a retina screen is not advised.

Reply:
Okay, but browserstack should still work, right. There, the website is rendered on real devices in real browsers which aren't dependent on your resolution, and this is then streamed to your retina screen.

Reply:
@carlobeltrame I'm not sure If this is resolved. Can you please check again?

Reply:
![Screenshot 2023-05-29 at 16-25-48 eCamp3](https://github.com/ecamp/ecamp3/assets/7566995/6035adc8-a4a6-4cfe-ab71-62494f15f17b)

The issue is still present. I noticed that if I remove the `vertical-align: middle` rule from the `.v-icon.v-icon` class, it looks as expected. (But I think Vuetify creates this rule and it's important for the application logo, so maybe override it for these icons?)

Reply:
Should be solved now

Reply:
Still not, no. Your new rule `.vertical-baseline` has lower specificity than the vuetify rule `.v-icon.v-icon`. Sorry to keep you up with this very slight detail, but you are really not in the dark here. Browserstack is intended exactly for this. Don't be afraid to use it :)

Reply:
Thanks, now it works :) will review the rest soon

Reply:
Sorry, this mistake has slipped through. Now I've tested it on Mac Chrome and Windows Firefox and it both correctly aligned. But I could not find a Desktop Linux on Browserstack.